src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -17,7 +17,7 @@ public class ApprenticeshipController : ControllerBase
         public async Task<IActionResult> AddApprenticeship(CreateApprenticeshipCommand request)
         {
             await _mediator.Send(request);
-            return Ok();
+            return Accepted();
```

Comment:
I agree we should change this.  I'm not sure it belongs in a PR about adding acceptance tests?  That makes it hard if someone comes along later to discover why this change was made.  Maybe it would be better to undo this change for this particular PR and create a separate, very simple PR to change the behaviour.